### PR TITLE
Updated READ.ME to include the updated METADATA options

### PR DESCRIPTION
**Pull request by Michaelgathara**

### CODE REVIEW

**Comment by Michaelgathara** [comment]
Done!

File: templates/local/s386/METADATA [hunk 1]
```diff
@@ -1,3 +1,4 @@
...
\ No newline at end of file
+SEO_IMAGE: https://github.com/Advocate-ABIC/advocateabic.github.io
+SEO_DESCRIPTION:
```
Comment (4shub) [inline templates/local/s386/METADATA:3]: Just take out line number 3

File: templates/local/s386/METADATA [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
\ No newline at end of file
+SEO_IMAGE: https://github.com/Advocate-ABIC/advocateabic.github.io
```
Comment (4shub) [inline templates/local/s386/METADATA:2]: Also, not sure if this is an image

Comment (Michaelgathara) [reply templates/local/s386/METADATA:2]: oof

**Pull request merged**